pandas/tests/scalar/period/test_period.py

```diff
@@ -767,6 +768,40 @@ def test_period_deprecated_freq(self):
             assert isinstance(p1, Period)
             assert isinstance(p2, Period)
 
+    def period_constructor(bound, offset):
```

Comment:
Maybe just define this in the test since it's only used there.

Reply:
OK done